lib/Icon/IconAccessibilitySupport.js

```diff
@@ -0,0 +1,30 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name IconAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['icon']}
+	],
+	
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (props) {
+			sup.apply(this, arguments);
+			// this.$.tapArea component is dynamically created using createComponents in smallChanged(), and call iconChanged(), so
+			// should check whether or not it exists.
+			if (this.$.tapArea) {
+				this.$.tapArea.set('accessibilityDisabled', this.accessibilityDisabled);
```

Comment:
Do you mean to propagate accessibilityDisabled or to always disable accessibility for the tapArea?